wallet/udb/txmined.go

```diff
@@ -2706,17 +2710,19 @@ func (s *Store) MakeInputSource(ns, addrmgrNs walletdb.ReadBucket, account uint3
 	// The simplest way to handle this is to branch to either cursor.First
 	// or cursor.Next depending on whether the cursor has already been
 	// created or not.
-	var bucketUnspentCursor, bucketUnminedCreditsCursor walletdb.ReadCursor
+	var bucketUnspentCursor, bucketUnminedCreditsCursor, bucketMultiSigCursor walletdb.ReadCursor
 
 	// Current inputs and their total value.  These are closed over by the
 	// returned input source and reused across multiple calls.
 	var (
-		currentTotal   dcrutil.Amount
-		currentInputs  []*wire.TxIn
-		currentScripts [][]byte
+		currentTotal      dcrutil.Amount
+		currentInputs     []*wire.TxIn
+		currentScripts    [][]byte
+		redeemScriptSizes []int
+		inputDetail       txauthor.InputDetail
```

Comment:
seems like we shouldn't have the `InputDetail` declared here, since it holds the same data as the other fields and can be created from them when we are ready to return.